sympy/calculus/util.py

```diff
@@ -1212,11 +1216,15 @@ def __div__(self, other):
                 if S.Zero not in other:
                     return self * AccumBounds(1/other.max, 1/other.min)
 
-                if S.Zero in self and S.Zero in other:
+                if (self.min.is_zero or self.max.is_zero ) and (other.min.is_zero or other.max.is_zero):
```

Comment:
`S.Zero in self` means that zero is between `self.min` and `self.max`. It does not mean that one of those should be zero.

Reply:
Oh yes , i'll commit again with the changes